tests/helpers/getAdvices.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { getAdvices } from '../../src/helpers/getAdvices';
+
+describe('test in getAdvices', () => { 
+
+  test('should first', async () => { 
+
+    const advice = await getAdvices();
+
+    expect(advice).toEqual({
+      id: expect.any(Number),
```

Comment:
Como el response siempre va a ser constante con base a tu mock en el handlers.js, puedes revisar expliciatmente si el "id" es "2" y el advice es igual a "Smile and the world smiles with you. Frown and you're on your own." justo como lo configuraste en el handlers.js :D

Reply:
Justo había ya hecho esos cambios. Si bien en el test lo dejé como me dijiste y la configuración también y todo funciona de manera correcta, en el archivo de `jest.setup.js` me marca que las funciones `beforeAll`, `afterEach` y` afterAll` no se encuentran definidas, pero al usarlas dentro del test, no me aparecen dichos errores. Preguntando a ChatGPT me dice lo siguiente: 

![image](https://github.com/AlanPinhon/advice-generator-app-ts/assets/74801980/c9505d23-20b2-4f53-aba2-e1d56130a39b)

Reply:
Podrías hacer un import del beforeAll, afterEach y afterAll desde vitest o jest y dejarlo en el jest.setup.js. Esto normalmente se hacer para no repetir código. Imagina que tienes que hacer 60 tests y poner en cada archivo dicho setup. Adicionalmente, ¿Qué pasaría si `server` cambia a otro nombre? Tendrías que cambiar esos 60 archivos en vez de uno solo. 🎃 

Para este proyecto tal vez no haya inconveniente alguno, pero para un proyecto grande no creo que sea un buen approach repetir código